client/state/selectors/is-selected-site-domain-only-site.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { isDomainOnlySite, getSelectedSiteId } from 'state/selectors';
+
+/**
+ * Returns true if selected site is a Domain-only site, false if the site is a regular site,
+ * or null if there is no selected or site is unknown.
+ *
+ * @param  {Object}   state  Global state tree
+ * @return {?Boolean}        Whether selected site is a Domain-only site
```

Comment:
It is valid [as far as I can tell
](http://usejsdoc.org/tags-type.html)

![screen shot 2017-01-31 at 13 48 46](https://cloud.githubusercontent.com/assets/326402/22463772/3774d420-e7bc-11e6-837d-e74d24ead44a.png)